python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -104,5 +116,12 @@ def _build_treelite_classifier(m, data, arg={}):
 
 
 def _treelite_fil_accuracy_score(y_true, y_pred):
-    y_pred_binary = y_pred > 0.5
-    return cuml.metrics.accuracy_score(y_true, y_pred_binary)
+    """Function to get correct accuarcy for FIL (returns probabilities)"""
+    y_pred_binary = input_utils.convert_dtype(y_pred > 0.5, np.int32)
+    if isinstance(y_true, np.ndarray):
+        return cuml.metrics.accuracy_score(y_true, y_pred_binary)
+    elif cuda.devicearray.is_cuda_ndarray(y_true):
+        y_true_np = y_true.copy_to_host()
+        return cuml.metrics.accuracy_score(y_true_np, y_pred_binary)
+    else:
+        return cuml.metrics.accuracy_score(y_true, y_pred)
```

Comment:
I don't understand this third case... maybe a comment? If y_true is neither a numpy array nor a gpu array, then we don't need to binarize y_pred?

Reply:
Fixed, we should use `y_pred` for this case as well